src/Components/ActionCard/ActionCard.scss

```diff
@@ -1,6 +1,9 @@
-@import '../../styles/Colors';
+@import '../../styles/Colors.scss';
 
 .card-header, .card-footer {
     background-color: $white;
     border:none;
 }
+.fa-pencil {
+    color:orange;
```

Comment:
color needs to be from colors.scss

Reply:
Also color is not general color like orange yellow red, refer the Optus std colors and choose those only